### ISSUE TITLE
Worker interrupted with signal: sigkill

### ISSUE THREAD

**Issue by benjy** [motiz88/flow-docker#7]
I'm running into this issue when running the container, my flowconfig works fine locally on the host.

```
docker run --rm -it -v $(pwd):/app motiz88/flow
Worker interrupted with signal: sigkill
Fatal error: exception End_of_file
```

I've tried server.max_workers=1 after some googling but that didn't help. Do you have any other suggestions?

**Comment by motiz88** [comment]
Nope, sorry. Flow has gone through significant changes since I created this repo, which is now years out of date; there are also official Flow binaries for Windows now, so the original reason for having this repo doesn't exist anymore.

Out of curiosity - what is your use case for running Flow inside Docker?

**Comment by benjy** [comment]
Locally I don't normally but i'm trying to get it running on Gitlab CI, where I'm using docker in docker, so everything has to run in a container.
